doc/tutorials/gaussian_transformation.ipynb

```diff
@@ -0,0 +1,298 @@
+{
```

Comment:
<p>he state with no photons in the wire.  -&gt; Just check with the others: I think there could be vacuum fluctuations, so even though the average photon number is zero there might be photons in the wire? It's a bit subtle, that question.</p>

 _Reply via <a href='https://app.reviewnb.com/XanaduAI/pennylane/pull/218/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>I agree. Its safer to write "average photon number is zero". </p>